pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -491,6 +491,35 @@ func Convert_v1_FeatureHyperv_To_api_FeatureHyperv(source *v1.FeatureHyperv, hyp
 	return nil
 }
 
+// This function parses the SRIOV-VF-PCI-ADDR variable that is set by SR-IOV
+// device plugin listing PCI IDs for devices allocated to the pod. The format
+// is as follows:
+//
+// "": for no allocated devices
+// "0000:81:11.1,": for a single device
+// "0000:81:11.1,0000:81:11.2[,...]": for multiple devices
+func getSRIOVPciAddresses() []string {
+	pciAddrString, isSet := os.LookupEnv("SRIOV-VF-PCI-ADDR")
```

Comment:
Do we know why this variable is exposed in the first place? Good for our use-case, but it looks like the device plugin provides a proper setup interface already ...